### PR TITLE
fix(a11y): file upload button focus visible

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/sass/components/ui/file-upload.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#file-upload {
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/sass/components/ui/file-upload.scss:1]: Might be good to give the elements more specific class names and maybe even put it into the component just to reduce risk of any CSS contamination.

File: packages/forms-web-app/src/views/pages/examination/select-file.njk [hunk 2]
```diff
@@ -154,8 +154,8 @@
 				{% endif %}
 			>
 				const target = document.documentElement;
-				const dropZone = document.querySelector('.dropZone');
-				const fileInput = document.getElementById('file-upload')
+				const dropZone = document.querySelector(".dropZone");
+				const fileInput = document.getElementById("file-upload");
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/views/pages/examination/select-file.njk:158]: Quotes correcting to double? Is this right? I think mine are single.

Comment (jonathangoulding) [reply packages/forms-web-app/src/views/pages/examination/select-file.njk:158]: passing linter ?

Comment (DavidBeale) [reply packages/forms-web-app/src/views/pages/examination/select-file.njk:158]: not a JS file - so prob no linting 🤔

Comment (jonathangoulding) [reply packages/forms-web-app/src/views/pages/examination/select-file.njk:158]: 'fixed'

File: packages/forms-web-app/src/sass/components/ui/file-upload.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.document-file-upload {
+	&:focus {
+		+ .govuk-button {
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/sass/components/ui/file-upload.scss:3]: This could become `.document-file-upload__label`

Comment (christopher-semmens) [reply packages/forms-web-app/src/sass/components/ui/file-upload.scss:3]: Apologies, I meant that the label element with the `.govuk-button` class could have another class added to the element which would be `.document-file-upload__label`. How the input was defined before was fine.

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**